rfcs/0000-commonmark-docs.md

```diff
@@ -0,0 +1,334 @@
+---
+feature: commonmark-docs
+start-date: 2020-07-05
+author: mboes
+shepherd-team: garbas, zimbatm, Kloenk
+shepherd-leader: Infinisil
+---
+
+# Summary
+[summary]: #summary
+
+Nix, Nixpkgs and NixOS documentation is currently in Docbook format.
+We propose to migrate all existing content to CommonMark, a flavour of
+Markdown.
+
+# Motivation
+[motivation]: #motivation
+
+Documentation ought to be easy to write, easy to maintain, easy to
+make pretty, and its format be so boring as to seldom be the object of
+future RFC's. But beyond that, the significance of the documentation
+format is in its very real impact on the perception of the project.
+Documentation is the storefront of any project online in addition to
+its front page. Modern looking documentation that can be edited at the
+click of a button sends a signal: that the Nix* projects are forward
```

Comment:
Excellent proof why this is _urgently_ necessary: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+%22Docs%3A+dockerTools%22

Reply:
And ventilating a little further my **_frustration_** of not having been able to get a doc improvement merged myself within due effort: https://discourse.nixos.org/t/dockertools-imprecisions/8770/3